crates/bevy_pbr/src/light.rs

```diff
@@ -41,9 +41,15 @@ use crate::{
 #[derive(Component, Debug, Clone, Copy, Reflect)]
 #[reflect(Component, Default)]
 pub struct PointLight {
+    /// Color of the light emitted.
     pub color: Color,
+    /// Amount/strength of light emitted, in lumens.
     pub intensity: f32,
+    /// How far light extends from the surface (TODO: or center?) of the point light.
     pub range: f32,
+    /// The size (radius) of the point light itself.
+    ///
+    /// A radius of 0.0 (the default) is a point with no measureable size, while a radius of 1.0 is a unit sphere, etc.
```

Comment:
I don't know if it's the case in Bevy, but usually the size of the light source affects the sharpness of the shadows. The bigger the light source (keeping the light source at the same distance from the object), the softer the shadows get as they are further from the object. I mention the distance, because the sun is a huge sphere, but since it's so far it works as a directional light giving mostly sharp shadows (?).

Might be interesting to mention?

Rendered in Blender:

Light radius: 0.1m
![image](https://user-images.githubusercontent.com/188612/205045839-bf2bd029-4f35-4eae-8f6c-422ae58fbc9d.png)

Radius: 1m
![image](https://user-images.githubusercontent.com/188612/205046087-d59b0c20-1816-41e2-8a1f-a466e445bf7f.png)

Radius: 2m
![image](https://user-images.githubusercontent.com/188612/205046200-e86e5e4e-c773-490b-afaa-7302c76a362d.png)

Reply:
There are no soft shadows yet. This would be addressed by something like #3631 .